cli/cli.go

```diff
@@ -136,7 +136,12 @@ func preRun(cmd *cobra.Command, args []string) {
 	configFile := configuration.Settings.ConfigFileUsed()
 
 	// initialize inventory
-	inventory.Init(configuration.Settings.GetString("directories.Data"))
+	err := inventory.Init(configuration.Settings.GetString("directories.Data"))
+	if err != nil {
+		// feedback.Errorf("Invalid option for --log-level: %s", configuration.Settings.GetString("logging.level"))
```

Comment:
Was this comment intended? It doesn't seem relevant.

Reply:
Nope, that should have been deleted. Thanks for catching that!